plugin/src/steps/source-nodes/create-nodes/create-remote-file-node/index.js

```diff
@@ -371,7 +378,9 @@ module.exports = ({
   ext = null,
   name = null,
   reporter,
+  pluginOptions,
 }) => {
+  const limit = pluginOptions?.type?.MediaItem?.localFile?.requestConcurrency
```

Comment:
i guess we technically don't need optionals here, since we're setting defaults right?

Reply:
We might since a user could potentially set type.MediaItem null 🤔 unless Joi is preventing that from happening

Reply:
ohh right, good point

Reply:
better safe than sorry!